source/features/one-click-diff-options.tsx

```diff
@@ -1,68 +1,12 @@
 import React from 'dom-chef';
 import select from 'select-dom';
-import delegate, {DelegateEvent} from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
 import {BookIcon, CheckIcon, DiffIcon, DiffModifiedIcon} from '@primer/octicons-react';
 
 import features from '../feature-manager';
-import attachElement from '../helpers/attach-element';
 import observe from '../helpers/selector-observer';
 import {removeTextNodeContaining} from '../helpers/dom-utils';
 
-const diffSwitchButtons = features.getIdentifiers(import.meta.url);
-
-function alternateDiffNatively(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): void {
-	const type = new URLSearchParams(event.delegateTarget.search).get('diff')!;
-	const formField = select(`input#diff_${type}`);
-	if (!formField) {
-		// Let the link through
-		return;
-	}
-
-	// Submit form so that the preference is persisted #5288
-	formField.checked = true;
-	formField.form!.submit();
-	event.preventDefault();
```

Comment:
My previous solution: 😱  catch the links and submit the existing form
My new solution: 😌 just create the form